bundles/edu.kit.kastel.sdq.eclipse.common.core/src/edu/kit/kastel/sdq/eclipse/common/core/model/StackingPenaltyRule.java

```diff
@@ -0,0 +1,70 @@
+/* Licensed under EPL-2.0 2022. */
+package edu.kit.kastel.sdq.eclipse.common.core.model;
+
+import java.util.List;
+
+import org.eclipse.jgit.annotations.Nullable;
+
+import com.fasterxml.jackson.databind.JsonNode;
+
+import edu.kit.kastel.sdq.eclipse.common.api.model.IAnnotation;
+
+public class StackingPenaltyRule extends PenaltyRule {
+
+	private static final String DISPLAY_NAME = "Stacking Penalty";
+	public static final String SHORT_NAME = "stackingPenalty";
+
+	// Penalty stored with decimal-point shifted one to the right (make sure no
+	// rounding issues happen)
+	private int penalty;
+
+	@Nullable
```

Comment:
I'm not sure about the Nullable annotation from JGit.

Reply:
I removed it.